automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/metadata/DataSpec.scala

```diff
@@ -66,58 +66,63 @@ class DataSpec extends FreeSpec with WebBrowserSpec
 
   "Writer and reader should see new columns" - {
     "with no defaults or local preferences when analysis run that creates new columns" in withWebDriver { implicit driver =>
-      val owner = UserPool.chooseProjectOwner
-      val reader = UserPool.chooseStudent
+      val Seq(owner, reader) = Seq(UserPool.chooseProjectOwner, UserPool.chooseStudent)
```

Comment:
Nit: I'm not convinced this is more readable than the way it was. :man_shrugging: